linux_os/guide/system/permissions/restrictions/sysctl_kernel_core_pattern_empty_string/bash/shared.sh

```diff
@@ -0,0 +1,60 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle
```

Comment:
Not sure if the remediation should have Platform similar to prodtype, e.g. rhel9 only.

Reply:
If a rule has specified prodtype in the rule.yml, you usually want to put multi_platform_all in the remediation scripts, because the platform is already defined by the prodtype. The exception is that you are sure that the remediation won't work on some of the products specified by prodtype.

Reply:
I'm going to switch this to `multi_platform_all`